.github/workflows/publish.yml

```
@@ -9,9 +9,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v3
         with:
-          node-version: '8'
+          node-version: '18'
```

Lag gjerne ei `.nvmrc`-fil med `18` i , og bruk her:

```suggestion
          node-version-file: '.nvmrc'
```

Veldig digg for oss som brukar nvm!

---

Nice! Kan du gjøre sånn ja. Det er kult. 8b78ed6